locales/en.yml

```diff
@@ -186,6 +186,10 @@ en:
           Please visit access.redhat.com to download client binaries.
         service_not_enabled: |-
           '%{service}' service is not enabled.
+        kube_not_supportted: |-
+          Client binary installs for kubernetes service is not supported yet.
+          Please refer http://kubernetes.io/docs/getting-started-guides/docker/#download-kubectl and
+          download the appropriate binary.
```

Comment:
Even if user downloads the binary, we do not have support for out of the box kubectl CLI binary connection to kube API server running inside the box. Probably we need to just display the message
"Client binary install for kubernetes service is not support yet."